Packs/EWS/Integrations/EWSv2/EWSv2.py

```diff
@@ -1137,8 +1137,11 @@ def parse_incident_from_item(item, is_fetch):
                         if hasattr(attachment, 'item') and attachment.item.mime_content:
                             attached_email = email.message_from_string(attachment.item.mime_content)
                             if attachment.item.headers:
-                                attached_email_headers = [(h, ' '.join(map(str.strip, v.split('\r\n')))) for (h, v) in
-                                                          attached_email.items()]
+                                attached_email_headers = []
+                                for h, v in attached_email.items():
+                                    if hasattr(v, 'split'):
```

Comment:
check that the type is `string` instead otherwise it may also crash due to the `str.split`